.gitignore

```diff
@@ -18,6 +18,7 @@ out
 # make sure the external config files are built 
 # on user's side each time, instead of using the ones from repo
 luthier/config/external
```

Comment:
If we're not going to be using this directory anymore, we can
also remove it from the .gitignore. That's not big deal though.

Reply:
What I changed it to is that we ignore the current *Config.json files one by one, but not the entire dir. 
This way we are a little bit more explicit when we think some config files should always be rebuilt, by adding that to the .gitignore list.